docs/tutorial/new-tutorial.md

```diff
@@ -0,0 +1,384 @@
+(tutorial)=
+# SQLair tutorial
+
+Imagine you are writing a Go program that handles your customers and their
+orders. The business logic is contained in the Go program but the customer and
+order data is stored in a SQL database. In this tutorial, we'll see how easy
+SQLair makes it to map the database between the database and the Go program.
+Let's get started!
+
+
+```{note}
+This tutorial will take around 30 minutes to complete. If you want a quicker
+start see {ref}`prepare-and-run`.
+```
+
+## Setting up the environment
+
+To start with create a new folder called tutorial and initialise a new Go project.
```

Comment:
```suggestion
To start with, create a new folder called `tutorial` and initialise a new Go project.
```

Reply:
In fact, I'd simplify even further:

```suggestion
The first thing you'll need is a Go project. Let's create one:
```

Then I'd add line by line comments in the code block, just to help someone who's really clueless catch up.